README.md

```diff
@@ -61,6 +61,12 @@ Using docker image.
 $ docker run --rm -v $PWD:/work k1low/tbls doc postgres://dbuser:dbpass@hostname:5432/dbname
 ```
 
+In SSL mode, add "?sslmode=disable"
```

Comment:
I think you need to disable ssl mode when the PostgreSQL server does not support SSL connections.

Reply:
Sorry, I didn't explain enough.
I just would like you to add this statement, not replace.

So, This is what I meant.

Before:
![image](https://user-images.githubusercontent.com/20323093/118370711-f35b9080-b5e3-11eb-872f-aba23184bae0.png)

After:
![image](https://user-images.githubusercontent.com/20323093/118370718-f9ea0800-b5e3-11eb-896b-47a061077aef.png)

But, I made a mistake first pull request.
I'm sorry for confusing you.

I tried to add the statement under PostgreSQL connection.
But, first pull request has an error, I added the statement under Quick Start.

So, I fixed it and sent you pull request again.

Reply:
Why `In SSL mode, add "?sslmode=disable"` ?

SSL mode is a connection method between client and server. It is not a setting that should always be disabled.

Reply:
Sorry, my phrasing was bad.
I will rephrase it.

## Before:
```
In SSL mode, add "?sslmode=disable"
```

## After:
```
When you want to disable SSL mode, add "?sslmode=disable"
```